feature/sessions/src/test/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenTest.kt

```diff
@@ -79,6 +79,17 @@ class SearchScreenTest {
         }
     }
 
+    @Test
+    @Category(ScreenshotTests::class)
+    fun checkEmptyBodyShot() {
+        searchScreenRobot {
+            setupSearchScreenContent()
+            inputDummyTextSearchTextFieldAppBar()
+            checkExistEmptyBody()
```

Comment:
May I ask what we check here?
I thought we could spot change in the screen image easily. 👀

Reply:
@takahirom 
This is a test to see if EmptyBody is displayed when a session is not found. 💁‍♂️

Reply:
I mean while checkScreenCapture() takes a screenshot of the screen, what checkExistEmptyBody() will check? I was thinking we could remove this 👀